build.gradle

```diff
@@ -1,43 +1,100 @@
-apply plugin: 'java'
-apply plugin: 'application'
-apply plugin: 'groovy'
+import java.time.Instant
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
 
-sourceCompatibility = 1.8
-mainClassName = "uk.gov.justice.digital.licences.pdf.Server"
+buildscript {
+    ext {
+        springBootVersion = '2.1.5.RELEASE'
+    }
+    repositories {
+        jcenter()
+        mavenCentral()
+    }
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
+        classpath 'com.github.ben-manes:gradle-versions-plugin:0.21.0'
+    }
+}
 
-version = 'v0.1.' + (System.getenv("CIRCLE_BUILD_NUM") ?: "SNAPSHOT")
+apply plugin: 'java'
+apply plugin: 'org.springframework.boot'
+apply plugin: 'io.spring.dependency-management'
+apply plugin: 'groovy'
+apply plugin: 'com.github.ben-manes.versions'
 
```

Comment:
gradle seems to prefer using a plugin block over `apply plugin` now: https://docs.gradle.org/current/userguide/plugins.html#sec:old_plugin_application